src/main/java/ch/njol/skript/util/slot/ItemFrameSlot.java

```diff
@@ -48,6 +48,14 @@ public void setItem(@Nullable ItemStack item) {
 		frame.setItem(item);
 	}
 	
+	@Override
+	public int getAmount() {
+		return 1;
```

Comment:
Considering the behaviour of the item amount syntax, shouldn't this properly set the item's amount?

Reply:
Item frames can only hold an item stack with size 1;